zerver/tests/test_signup.py

```diff
@@ -1299,6 +1299,14 @@ def test_invite_user_to_realm_on_manual_license_plan(self) -> None:
             result, "All Zulip licenses for this organization are currently in use"
         )
 
+        with self.settings(BILLING_ENABLED=True):
+            result = self.invite(
+                self.nonreg_email("bob"),
+                ["Denmark"],
+                invite_as=PreregistrationUser.INVITE_AS["GUEST_USER"],
```

Comment:
This kind of stuff can be cleaned up in a follow-up i think, as I mentioned in https://chat.zulip.org/#narrow/stream/3-backend/topic/PreregistrationUser.2EINVITE_AS since all this stuff is `PreregistrationUser.INVITE_AS["..."]` and `UserProfile.ROLE_...` being used interchangably

Reply:
Makes sense.